src/cadenza.js

```diff
@@ -64,6 +64,14 @@ globalThis.cadenza = Object.assign(
  * _Note:_ The GeoJSON geometry type "GeometryCollection" is currently not supported.
  */
 /** @typedef {[number,number,number,number]} Extent - An array of numbers representing an extent: [minx, miny, maxx, maxy] */
+/** @typedef {'normal'|'simplified'} OperationMode - The mode in which a workbook should be operated */
+/**
+ * @typedef {'workbook-design'|'workbook-view-management'} UiFeature - The name of a Cadenza UI feature
+ *
+ * _Note:_ Supported features are:
+ * * 'workbook-design' - Disable the designer
+ * * 'workbook-view-management' - Disable workbook layout/design editing
```

Comment:
Maybe add the hint from the sprint review? "Is included in 'workbook-design'."

Reply:
Added :)